Aptoide/aptoide-client-v8
# [MOB-478] Install in background android 10

**What does this PR do?**

   This PR aims at fixing the installation flow for when the app is in background on Android 10. Since we were on the background we would not be able to show the install dialog. For this reason, we had to make a few changes (visual and flow wise).

**Database changed?**

   Yes

**Where should the reviewer start?**


**How should this be manually tested?**

  Download 2 apps ( you can put it through apps - 2 updates) close the app and check that after everything is finished downloading you will have a notification indicating you that some apps are ready to be installed. If you click on it, it will lead you to apps.
This flow also works in case you only download one app. On that case, you should receive a notification indicating that only the downloaded app is ready to install and if you click on it, it will lead you to the appview and prompt the install dialog without you having to click on the install button.

**What are the relevant tickets?**

  Tickets related to this pull-request: [MOB-478](https://aptoide.atlassian.net/browse/MOB-478)

**Questions:**

   Does this add new dependencies which need to be added? (Eg. new keys, etc.) 



**Code Review Checklist**
